engines/director/lingo/lingo-codegen.cpp

```diff
@@ -274,22 +274,20 @@ int Lingo::codeFloat(double f) {
 	return _currentScript->size();
 }
 
-int Lingo::codeConst(int val) {
-	int res = g_lingo->code1(g_lingo->c_constpush);
+int Lingo::codeInt(int val) {
```

Comment:
Why did you change the implementation?

Reply:
Originally codeConst() would push a c_constpush instruction and then an encoded integer to the current script. As far as I could tell there was no shortcut for pushing only the encoded integer to the script, something which needs to be done quite a lot, so I adjusted codeConst to not add the push instruction, similar to how the other codeX shortcuts work. I re-added the push instruction to the few places codeConst was originally called. I've still yet to go and replace all of the longhand extra WRITE_UINT32 calls in lingo-gr.y with codeInt()